### DISCUSSION TITLE
在IT之家开启字体比例缩放会造成评论区不显示

### DISCUSSION THREAD

**Discussion by Xia0xia0Xia0** [F9y4ng/GreasyFork-Scripts#263] [问答专区(Question / Answer)] [answered]
## 关闭字体渲染时正常的状态：
![正常的](https://github.com/F9y4ng/GreasyFork-Scripts/assets/101876873/0055cbdb-561f-4b95-a2b0-473c7b865ef4)

## 开启字体渲染字体缩放1.250时有问题的状态：
![有问题的](https://github.com/F9y4ng/GreasyFork-Scripts/assets/101876873/6a8f86bb-0efb-4383-8f64-24b3e9624721)

应该如何修复它？

我的本地环境是 WIN10,CHROME 117,TM4.19.0

**Comment by F9y4ng** [comment, marked as answer]
这个问题已经在新的月度更新中修复了，新版本在本地测试没有问题后，会在下个月更新中发布。

如果你等不了需要手动修复该问题，请找到以下代码：
https://github.com/F9y4ng/GreasyFork-Scripts/blob/20f1254eda8a0e55c9e2e29cbff04fc23e664b73/Font%20Rendering.user.js#L2968-L2971

并在predefinedSitesProps中增加一个新的值：
```js
"www.ithome.com": { Element: ["scrollHeight"] }, 
```
保存代码，刷新页面。（在TM4.19.0中，本地修改代码的操作会使自动更新失效，需要手动恢复）

---

BTW. 在新更新中，已经将这个内部数据提取至外部可编辑了。

**Reply by Xia0xia0Xia0** [reply]
非常感谢！
